OSDOCS-16950#CQA work for Stor6 - LSO (#115348)
openshift/openshift-docs: modules/persistent-storage-local-overview.adoc

// Module included in the following assemblies:
//
// * storage/persistent_storage/persistent-storage-local.adoc

:_mod-docs-content-type: CONCEPT
[id="local-storage-overview_{context}"]

> = Local Storage Operator overview

Reviewer: Not sure this needs its own module....

[role="_abstract"]

> Local volumes can be used without manually scheduling pods to nodes because the system is aware of the volume node constraints. However, local volumes are still subject to the availability of the underlying node and are not suitable for all applications.

Reviewer: Remove the passive? Is this still accurate?

```suggestion
You can use local volumes without manually scheduling pods to nodes because the system is aware of the volume node constraints. However, local volumes are still subject to the availability of the underlying node and are not suitable for all applications.
```
